### PR TITLE
ceph.spec: include ninja as BuildRequires

### PR DESCRIPTION
**Pull request by tchaikov**
to ease the build for developers using SUSE, Fedora, CentOS or RHEL.
so install-deps.sh can install ninja for them.

Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: ceph.spec.in [hunk 1]
```diff
@@ -350,6 +350,7 @@ BuildRequires:	lz4-devel >= 1.7
 %if 0%{with make_check}
 %if 0%{?fedora} || 0%{?rhel}
 BuildRequires:	libtool-ltdl-devel
+BuildRequires:	ninja
```
Comment (rishabh-d-dave) [inline ceph.spec.in:353]: Failed for me on Fedora 33. Changing `ninja` to `ninja-build` fixed it.

**Review by rishabh-d-dave — CHANGES_REQUESTED** [review]

File: ceph.spec.in (hunk 1 above)
Comment (tchaikov) [reply ceph.spec.in:353]: @rishabh-d-dave fixed and repushed.

**Review by rishabh-d-dave — APPROVED** [review]
Works now.

**Comment by rishabh-d-dave** [comment]
@tchaikov Should I close #39990?

**Pull request merged**